libraries/freertos_plus/standard/https/src/iot_https_client.c

```diff
@@ -109,13 +110,23 @@
  */
 #define HTTPS_PARTIAL_HOST_HEADER_LINE          HTTPS_HOST_HEADER HTTPS_HEADER_FIELD_SEPARATOR HTTPS_END_OF_HEADER_LINES_INDICATOR   
 
+/**
+ * @brief The string length of the maximum file size that can be represented in a 32 bit system.
+ * 
+ * If a number in a 32 bit system cannot be more than 32 bits, then 2^32 = 4294967296 which is 10 digits in length.
+ * This is used for local string array initialization.
+ */
+#define HTTPS_MAX_FILE_SIZE_LENGTH              ( 10 )
```

Comment:
it doesn't seem this is being used. Anyway, whats the need for defining this? Is there any restriction in the library that it can't download a file thats greater than this length ?

Reply:
I keep putting it back when I use it temporarily. 
I made the restriction anyways since the contentLength is a uint32_t and that is the maximum number than can be represented.